src/api/parser/src/utils/html/sanitize.js

```diff
@@ -50,10 +51,34 @@ module.exports = (dirty) => {
     allowedIframeHostnames: [
       'www.youtube.com',
       'player.vimeo.com',
+      'player.twitch.tv',
       'giphy.com',
       'cdn.embedly.com',
       'open.spotify.com',
       'medium.com',
     ],
+    transformTag: {
+      iframe: (tagName, attribs) => {
+        const swapParent = (parent, host) => {
```

Comment:
`parent` parameter is not used. It's better to name this `src` and use it in line 64

Reply:
All good!